guide/13-managing-arcgis-applications/enterprise-sites-guide.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>To add a new card, you must update the cards list for the row within the section with the dictionary for the new card. </p><p>First, we will access the row we want to add the new card to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>